tests/test_conf.py

```diff
@@ -399,6 +399,38 @@ def test_edit():
     assert lines == expected_lines
 
 
+def test_edit_with_include(tmp_path):
+    from pgtoolkit.conf import parse
+
+    content = "a = 1\n"
+    include_conf = tmp_path / "included.conf"
+    with include_conf.open("w") as f:
+        f.write(content)
+
+    main_conf = tmp_path / "somefile.conf"
+    content = "include 'included.conf'\n"
+    content += "#a = 2\n"
+    with main_conf.open("w") as f:
+        f.write(content)
+
+    conf = parse(str(main_conf))
+
+    with conf.edit() as entries:
+        entries["a"].value = 3
```

Comment:
It looks quite odd that "a" appears in `entries` but editing its value has no effect.

So I think we should rather filter `entries` returned by `edit()` from items not corresponding to a line in `conf` file.

Reply:
I agree with you. I tried to filter the entries in https://github.com/dalibo/pgtoolkit/blob/master/pgtoolkit/conf.py#L650 
But quickly faced other issues that I didn't manage to fix without a lot of changes.

Reply:
There are indeed other issues, I'll take a look at these so that you can follow-up with this change easier.